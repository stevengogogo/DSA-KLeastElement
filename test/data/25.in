10 50000
-6670 8645 -9300 8527 9076 -811 -1104 -6852 -8699 1015
Query 3 7 3
Delete 3
Insert 4 -8161
Delete 3
Insert 5 38
Delete 1
Delete 1
Reverse 1 4
Query 1 1 1
Delete 3
Insert 3 -2227
Query 5 6 2
Delete 1
Reverse 2 3
Delete 5
Delete 3
Reverse 5 5
Insert 6 -3641
Query 6 6 1
Insert 5 4504
Delete 3
Reverse 2 5
Delete 6
Insert 2 -7545
Reverse 5 6
Query 3 4 2
Insert 7 6276
Insert 4 5341
Delete 7
Insert 4 308
Query 4 5 1
Delete 8
Reverse 3 5
Query 2 4 3
Delete 6
Insert 6 -508
Delete 1
Query 3 5 2
Reverse 2 2
Query 4 6 3
Insert 7 -5292
Delete 4
Insert 3 -3753
Query 2 3 2
Delete 6
Insert 6 5647
Insert 2 471
Reverse 6 8
Delete 4
Reverse 2 4
Delete 7
Reverse 5 6
Query 6 6 1
Delete 1
Delete 1
Insert 5 -216
Query 5 5 1
Query 3 5 1
Delete 3
Query 3 3 1
Reverse 1 2
Insert 2 8381
Query 1 3 1
Query 1 4 2
Delete 1
Reverse 1 3
Query 2 2 1
Delete 1
Reverse 1 2
Delete 2
Query 1 2 2
Reverse 1 2
Reverse 1 2
Delete 1
Query 1 1 1
Delete 1
