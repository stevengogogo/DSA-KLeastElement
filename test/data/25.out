-811
-811
-1104
-3641
4504
308
5341
308
1015
5341
-5292
-216
-5292
-5292
471
471
5341
8381
-216
